cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc

```diff
@@ -0,0 +1,19 @@
+[id='creating-a-single-component-application-with-odo']
+= Creating a single-component application with {odo-title}
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: creating-a-single-component-application-with-odo
+toc::[]
+
```

Comment:
We cannot add links to modules. Please add all the prerequisites for all modules in the assembly in the assembly itself. 
I suggest we add a sentence as an introduction in this assembly (on the lines of - You can use odo to create a single component application, modify it, and connect it to other services as follows:) and then follow it up with the prerequisites section. 
Suggest you do this for all the assemblies, especially, if we need to add links.
Please add a link to the service catalog docs in OCP when you include that as a prerequisite in the assembly.